web/packages/teleport/src/Discover/Database/EnrollRdsDatabase/AwsRegionSelector.tsx

```diff
@@ -15,95 +15,68 @@
-import { requiredField } from 'shared/components/Validation/rules';
-import Validation, { Validator } from 'shared/components/Validation';
+import { Box, Text, Flex, ButtonSecondary, LabelInput } from 'design';
+import Select, { Option } from 'shared/components/Select';
 import { Refresh as RefreshIcon } from 'design/Icon';
 
 import { awsRegionMap, Regions } from 'teleport/services/integrations';
 
 export function AwsRegionSelector({
   onFetch,
   onRefresh,
-  disableFetch,
   disableSelector,
   clear,
 }: {
   onFetch(region: Regions): void;
   onRefresh(): void;
-  disableFetch: boolean;
   disableSelector: boolean;
   clear(): void;
 }) {
   const [selectedRegion, setSelectedRegion] = useState<RegionOption>();
 
-  function handleFetch(validator: Validator) {
-    if (!validator.validate()) {
-      return;
-    }
-    onFetch(selectedRegion.value);
-  }
-
   function handleRegionSelect(option: RegionOption) {
     clear();
     setSelectedRegion(option);
+    onFetch(option.value);
   }
 
   return (
-    <Validation>
-      {({ validator }) => (
-        <Box>
-          <Text mt={4}>
-            Select the AWS Region you would like to see databases for:
-          </Text>
-          <Flex alignItems="center" gap={3} mt={2} mb={3}>
-            <Box width="320px">
-              <FieldSelect
-                label="AWS Region"
-                rule={requiredField('Region is required')}
-                placeholder="Select a Region"
-                isSearchable
-                isSimpleValue
-                value={selectedRegion}
-                onChange={handleRegionSelect}
-                options={options}
-                isDisabled={disableSelector}
-              />
-            </Box>
-            <Flex alignItems="center">
-              <ButtonPrimary
-                disabled={disableFetch || !selectedRegion}
-                onClick={() => handleFetch(validator)}
-                width="160px"
-                height="40px"
-                mt={1}
-              >
-                Fetch Databases
-              </ButtonPrimary>
-              <ButtonSecondary
-                onClick={onRefresh}
-                ml={3}
-                mt={1}
-                title="Refresh database table"
-                height="40px"
-                width="30px"
-                css={`
-                  &:disabled {
-                    opacity: 0.35;
-                    pointer-events: none;
-                  }
-                `}
-                disabled={disableSelector || !disableFetch}
-              >
-                <RefreshIcon fontSize={3} />
-              </ButtonSecondary>
-            </Flex>
-          </Flex>
+    <Box>
+      <Text mt={4}>
+        Select the AWS Region you would like to see databases for:
+      </Text>
+      <Flex alignItems="center" gap={3} mt={2} mb={3}>
+        <Box width="320px" mb={4}>
+          <LabelInput htmlFor={'select'}>Aws Region</LabelInput>
```

Comment:
```suggestion
          <LabelInput htmlFor={'select'}>AWS Region</LabelInput>
```

AWS is the product name so it should be capitalised when displaying it to users

Reply:
Yeah I noticed it as well, but when I saw the actual page, they were capitalized, so I didn't mention them.
However, for the sake of consistency I think we should still capitalize correctly.